404.py

```diff
@@ -0,0 +1,28 @@
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    def sumOfLeftLeaves(self, root: TreeNode) -> int:
+        sum_left = 0
+
+        if not root:
+            return sum_left
+
+        queue = collections.deque([root])
```

Comment:
Since you are already using a queue, you can try to store some additional information into the queue to help you simplify line 22 and 23.